pkg/dependencymagnet/dependencymagnet.go

```diff
@@ -5,6 +5,7 @@
 package dependencymagnet
 
 import (
-	_ "github.com/jteeuwen/go-bindata/go-bindata"
-	_ "github.com/openshift/library-go/alpha-build-machinery"
+	_ "github.com/go-bindata/go-bindata"
+	_ "github.com/go-bindata/go-bindata/go-bindata"
```

Comment:
There seems to be an additional go-bindata entry here

Reply:
When I was testing this it needed both `github.com/go-bindata/go-bindata` and `github.com/go-bindata/go-bindata/go-bindata` to have both `make` and `make update` complete successfully.

I'm not certain why, as I'd assume the longer path would be sufficient.